discovery-provider/.test.env

```diff
@@ -1,4 +1,8 @@
 # TODO: dummy ganache keys for local setup; should wire with dynamically generated keys
 audius_delegate_owner_wallet=0x1D9c77BcfBfa66D37390BF2335f0140979a6122B
 audius_delegate_private_key=0x3873ed01bfb13621f9301487cc61326580614a5b99f3c33cf39c6f9da3a19cad
-audius_solana_rewards_manager_account=8MzNUaBHskteN7poTrZG5wgSNSbXQwieMDB4wk9fgB7f
\ No newline at end of file
+audius_solana_rewards_manager_account=8MzNUaBHskteN7poTrZG5wgSNSbXQwieMDB4wk9fgB7f
+
+audius_elasticsearch_url=http://localhost:9200
+audius_elasticsearch_run_indexer=true
+audius_db_url=postgresql+psycopg2://postgres:postgres@localhost/test_audius_discovery
```

Comment:
doesn't this need the port? `localhost:5432`

Reply:
i think 5432 is assumed as the default postgres port. it's copied from here. maybe we should be explicit everywhere? 
https://github.com/AudiusProject/audius-protocol/blob/2369a06e8f19535c9a0177af18ae121a3d4ab396/discovery-provider/integration_tests/conftest.py#L20

Reply:
updated them!